frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/numbered_list/numbered_list_icon.dart

```diff
@@ -19,26 +19,26 @@ class NumberedListIcon extends StatelessWidget {
   Widget build(BuildContext context) {
     final textStyleConfiguration =
         context.read<EditorState>().editorStyle.textStyleConfiguration;
-    final fontSize = textStyleConfiguration.text.fontSize ?? 16.0;
     final height =
         textStyleConfiguration.text.height ?? textStyleConfiguration.lineHeight;
     final combinedTextStyle = textStyle?.combine(textStyleConfiguration.text) ??
         textStyleConfiguration.text;
-    final size = fontSize * height;
-    return Container(
-      constraints: BoxConstraints(
-        minWidth: size,
-        minHeight: size,
-      ),
-      margin: const EdgeInsets.only(top: 0.5, right: 8.0),
-      alignment: Alignment.center,
-      child: Center(
-        child: Text(
-          node.levelString,
-          style: combinedTextStyle,
-          strutStyle: StrutStyle.fromTextStyle(combinedTextStyle),
-          textDirection: textDirection,
+    final adjustedTextStyle = combinedTextStyle.copyWith(height: height);
+
+    return Padding(
+      padding: const EdgeInsets.only(right: 8.0),
+      child: Text(
+        node.levelString,
+        style: adjustedTextStyle,
+        strutStyle: StrutStyle.fromTextStyle(combinedTextStyle),
+        textHeightBehavior: TextHeightBehavior(
```

Comment:
@LucasXu0 In testing, I find I didn't really need these configurations (constraints, top padding, center). Which should I add back?

Reply:
I can't recall either. Maybe you could try inputting a large number (10000).

Reply:
Ahh yes. That's a good suggestion. Here's the before and after. Changes made:

- don't center (see the difference between 8 and 16)
- fixed width numbers (see alignment in the text under the first point)


Before:
![Screenshot 2024-12-27 at 1 56 07 PM](https://github.com/user-attachments/assets/7c6f0adf-2c4b-4341-ae47-051837ad15f5)

After:
![Screenshot 2024-12-27 at 2 08 03 PM](https://github.com/user-attachments/assets/cd5b2605-6042-4488-96ed-99a290d05048)